.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,34 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
```

Comment:
Currently the default branch in [angelolab/ark-analysis](https://github.com/angelolab/ark-analysis/branches) and [angelolab/mibi-bin-tools](https://github.com/angelolab/mibi-bin-tools/branches) **master**. However for [angelolab/toffy](https://github.com/angelolab/toffy/branches), [angelolab/tmi](https://github.com/angelolab/tmi/branches) and [angelolab/cell_classification](https://github.com/angelolab/cell_classification/branches) the default branch is **main**. We can either:

1. Add `main` here in the `branches` section.
2. Change  the default branch from **master** to **main** for [angelolab/ark-analysis](https://github.com/angelolab/ark-analysis/branches) and [angelolab/mibi-bin-tools](https://github.com/angelolab/mibi-bin-tools/branches).

I'm partial to Option 2, as that's the standard being pushed forward.

Reply:
Agreed, makes sense